### DISCUSSION TITLE
ScriptReference/AsyncOperation-allowSceneActivation

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#149] [Notes]
# ScriptReference/AsyncOperation-allowSceneActivation

https://docs.unity3d.com/ScriptReference/AsyncOperation-allowSceneActivation.html

**Comment by vertxxyz** [comment]
This flag is ignored in `Awake` [by design](https://issuetracker.unity3d.com/issues/loadsceneasync-allowsceneactivation-flag-is-ignored-in-awake?_ga=2.223741080.284412013.1588492390-488124911.1555604428). Use `Start` or `Update` instead.

**Comment by docscomments** [comment]
**Comment by `@Baste-RainGames`:**  

Note that if this is false, yielding the AsyncOperation will never finish, as yielding checks the isDone variable:  

```csharp
var loadOp = SceneManager.LoadSceneAsync("scene");  
loadOp.allowSceneActivation = false;  
yield return loadOp; //never finishes!
```
